guides/common/assembly_installing-satellite-server-connected.adoc

```diff
@@ -6,7 +6,8 @@ When you install {ProjectServer} from a connected network, you can obtain packag
 
 NOTE: You cannot register {ProjectServer} to itself.
 
-Use the following procedures to install {ProjectServer}, perform the initial configuration, and import subscription manifests. For more information on subscription manifests, see {BaseURL}content_management_guide/managing_subscriptions/[Managing Subscriptions] in the _Content Management Guide_.
+Use the following procedures to install {ProjectServer}, perform the initial configuration, and import subscription manifests.
```

Comment:
This link does not work for the Satellite guide. It just reroutes to:

https://access.redhat.com/documentation/en-us/red_hat_satellite/6.8/html-single/content_management_guide/index#Managing_Subscriptions/

Reply:
That's because a  stupid slash at the end of the link, fixed now.